### PR TITLE
Structurize project according to the style guide

### PR DESCRIPTION
**Pull request by belgeu**
Since structuring of a project makes it a lot easier to navigate through project files, current project files were re-arranged to keep them in line with the following style guide - http://docs.python-guide.org/en/latest/writing/structure/ .

### CODE REVIEW

File: python/pip_packages.txt [hunk 1]
```diff
@@ -0,0 +1,2 @@
+###### PIP packages ######
```
Comment (extsoft) [inline python/pip_packages.txt]: File name has to be "requirements.txt".

File: python/tests/webdriver_test.py [hunk 2]
```diff
@@ -0,0 +1,21 @@
+import pytest
```
Comment (extsoft) [inline python/tests/webdriver_test.py]: Must be in `python/test_sample/`. In this package we have only unit tests.
The same about `python/tests/unit_test.py` file.

**Review by extsoft — CHANGES_REQUESTED** [review]

File: python/pip_packages.txt (hunk 1 above)
Comment (belgeu) [reply python/pip_packages.txt]: Re-named to "requirements.txt".

File: python/tests/webdriver_test.py (hunk 2 above)
Comment (belgeu) [reply python/tests/webdriver_test.py]: Re-placed to `python/test_sample/` directory.

**Review by extsoft — APPROVED** [review]

**Pull request merged**